ch_pipeline/processing/daily.py

```diff
@@ -242,19 +267,20 @@
         - "{catalogs[0]}"
         - "{catalogs[1]}"
         - "{catalogs[2]}"
+        - "{catalogs[3]}"
```

Comment:
The CGRaBS catalog is entirely contained within the OVRO catalog, so there's no reason to keep CGRaBS around anymore